source/guides/stacks/dlrs-inference.rst

```diff
@@ -19,7 +19,7 @@ Overview
 
 The solution covered here requires the following software components:
 
-* `Deep Learning Reference Stack`_ which is a |CL-ATTR| based Docker\* container providing deep learning frameworks and is optimized for Intel Xeon Scalable platforms.
+* `Deep Learning Reference Stack`_ which is a |CL-ATTR| based Docker\* container providing deep learning frameworks and is optimized for Intel® Xeon® platforms.
```

Comment:
```suggestion
* `Deep Learning Reference Stack`_ which is a |CL-ATTR| based Docker\* container providing deep learning frameworks and is optimized for the Intel® Xeon® platform.
```

Reply:
I believe we are good to use 'processors' per the sample usage chart provided in the names DB (which shows an example of 'processor' pluralized).